xpk.py

```diff
@@ -7265,6 +7516,79 @@ def directory_path_type(value):
 )
 workload_create_parser.set_defaults(func=workload_create)
 
+
+# "workload create-pathways" command parser.
+workload_create_pathways_parser = workload_subcommands.add_parser(
+    'create-pathways', help='Create a new job.'
+)
+workload_create_pathways_parser_required_arguments = (
+    workload_create_pathways_parser.add_argument_group(
+        'Workload create-pathways Built-in Arguments',
+        'Configure xpk to create a Pathways Workload for you.',
+    )
+)
+workload_create_pathways_parser_optional_arguments = (
+    workload_create_pathways_parser.add_argument_group(
+        'Optional Arguments',
+        'Arguments optional for `workload create-pathways`.',
+    )
+)
+workload_create_pathways_base_docker_image_arguments = workload_create_pathways_parser.add_argument_group(
+    'Base Docker Image Arguments',
+    'User supplies a base image or by default the image is set by xpk.'
+    ' Xpk will add the `script_dir` to the base image creating an anonymous'
+    ' docker image. These arguments are exclusive to `--docker-image`.',
+)
+workload_create_pathways_docker_image_arguments = workload_create_pathways_parser.add_argument_group(
+    'Docker Image Arguments',
+    '`--base-docker-image` is used by default. Set this argument if the'
+    ' user wants the docker image to be used directly by the xpk workload.',
+)
+
+### "workload create-pathways" Required arguments, specific to Pathways
+workload_create_pathways_parser_required_arguments.add_argument(
+    '--tpu-type',
```

Comment:
hmm a bit confused, why does pathways_parser for workload create need --tpu-type specifically when it is part of workload_create arguments as well

Reply:
workload create supports provision of one out of tpu-type or device-type in a mutually exclusive group (GPUs and CPUs are supported).
workload create-pathways can only support tpu-type.

These args are hence not added in the shared parsers command!